(FreeBSD) unable to build

[ 62%] Linking CXX executable ../../bin/graft-supernode
..
supernode/TxPool.cpp:(.text+0x435): undefined reference to `supernode::TxPool::m_rpc_timeout

--- a/src/supernode/TxPool.cpp
+++ b/src/supernode/TxPool.cpp
@@ -39,6 +39,7 @@ using namespace std;
 namespace supernode {
 
 TxPool::TxPool(const std::string &daemon_addr, const std::string &daemon_login, const std::string &daemon_pass)
+  : m_rpc_timeout(std::chrono::seconds(30))
 {
 
     bool result = false;

--- a/src/supernode/TxPool.h
+++ b/src/supernode/TxPool.h
@@ -56,7 +56,7 @@ protected:
 
 private:
     epee::net_utils::http::http_simple_client m_http_client;
-    static constexpr const std::chrono::seconds m_rpc_timeout = std::chrono::seconds(30);
+    std::chrono::seconds m_rpc_timeout;
 
 };
 

--- a/utils/build_scripts/android64.Dockerfile
+++ b/utils/build_scripts/android64.Dockerfile
@@ -37,7 +37,7 @@ ENV PATH $TOOLCHAIN_DIR/aarch64-linux-android/bin:$TOOLCHAIN_DIR/bin:$PATH
 
 ## Build BOOST
 RUN cd boost_${BOOST_VERSION} \
-    && ./b2 --build-type=minimal link=static runtime-link=static --with-chrono --with-date_time --with-filesystem --with-program_options --with-regex --with-serialization --with-system --with-thread --build-dir=android64 --stagedir=android64 toolset=clang threading=multi threadapi=pthread target-os=android stage
+    && ./b2 --build-type=minimal link=static runtime-link=static --with-chrono --with-date_time --with-filesystem --with-program_options --with-regex --with-serialization --with-system --with-thread  --with-locale  --build-dir=android64 --stagedir=android64 toolset=clang threading=multi threadapi=pthread target-os=android stage
 
 #INSTALL cmake (avoid 3.7 : https://github.com/android-ndk/ndk/issues/254)
 ENV CMAKE_VERSION 3.6.3
@@ -72,8 +72,8 @@ RUN curl -s -O https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz 
     && make build_crypto build_ssl \
     && cd .. && mv openssl-${OPENSSL_VERSION}  openssl
 
-RUN git clone https://github.com/monero-project/monero.git \
-    && cd monero \
+RUN git clone --single-branch --depth=1 https://github.com/graft-project/GraftNetwork.git \
+    && cd GraftNetwork \
     && mkdir -p build/release \
     && cd build/release \
     && CC=clang CXX=clang++ \

